pygmt/helpers/decorators.py

```diff
@@ -15,13 +15,14 @@
 COMMON_OPTIONS = {
     "R": """\
         region : str or list
```

Comment:
I *think* I might have figured it out, can you try the below? I.e. Use an r-string while removing the backslash and putting the `region : str or list` line right after the triple quotes.

```suggestion
    "R": r"""region : str or list
```